csharp/private/rules/binary_private.bzl

```diff
@@ -129,6 +130,9 @@ This is a private rule used to implement csharp_binary and should not be used di
             default = ":runtimeconfig.json.tpl",
             allow_single_file = True,
         ),
+        "internals_visible_to": attr.string_list(
+            doc = "C# libraries that can see internal our symbols. Using this rather than the InternalsVisibleTo assembly attribute will improve build caching.",
```

Comment:
```suggestion
            doc = "Other C# libraries that can see the assembly's internal symbols. Using this rather than the InternalsVisibleTo assembly attribute will improve build caching.",
```